bin/build-docker

```diff
@@ -6,18 +6,23 @@ export REPO=chanzuckerberg/idseq-web
 BRANCH="$1"
 COMMIT="$2"
 BUILD_NUMBER="$3"
-COMPOSE_TAG="chanzuckerberg/idseq-web:compose"
+COMPOSE_TAG="$REPO:compose"
```

Comment:
This script later got refactored out to https://github.com/chanzuckerberg/infra-tools/blob/master/docker-helper, which a bunch of teams use. That script includes a version of this optimization, but without the compose label.

Might be worth getting on that script version to see if it helps.

Usage example [here](https://github.com/chanzuckerberg/oauth2_proxy/blob/ea0160d7a50320fde398e785694f08242da98ccb/.travis.yml#L4), though make sure you use the latest version of the script from infra-tools.

Reply:
Ok that’s good to know, and independent validation of this change. 

How do you recommend we make use of that shared script? Copy it into our web repo? It looks like it takes care of pushing as well, so it could replace our dockerpush script.

Reply:
In discussion with Tiago, we decided to defer this work. See https://jira.czi.team/browse/IDSEQ-970.